File: projekat/AngularApp/Web2Projekat2/src/app/model/vehicle.ts
export class Vehicle
{
    public Id: number    
    public Model: string
    public Manufacturer: string
    public Year: number
    public Pictureurl: string
    public Description: string
    public CostPerHour: number
    public VehicleType: number
    public Branch: number
    public Reservations: object[]

  constructor()
  {
      this.Id;
      this.Model="";
      this.Manufacturer="";
      this.Year;
      this.Pictureurl="";
      this.Description="";
      this.CostPerHour;
      this.VehicleType;
      this.Branch;
      this.Reservations;
  }
}

File: projekat/RentApp/Models/Entities/Vehicle.cs
using System.Collections.Generic;

namespace RentApp.Models.Entities
{
    public class Vehicle
    {
        public int Id { get; set; }
        public string Model { get; set; }
        public string Manufacturer { get; set; }
        public int Year { get; set; }
        public string Pictureurl { get; set; }
        public string Description { get; set; }
        public double CostPerHour { get; set; }
        public VehicleType VehicleType { get; set; }
        public Service Branch { get; set; }
        public ICollection<Reservation> Reservations { get; set; }
    }
}